source/accounts/accounts_and_projects.rst

```diff
@@ -0,0 +1,251 @@
+
+**New to the NOAA RDHPCS Program?**
+
+Welcome!  The information below introduces how we structure user
+accounts, projects, and system allocations. It's all you need to know
+about getting to work. In general, NOAA RDHPCS resources are granted
+to projects in allocations on systems, and are made available to the
+users associated with each project.
+
+
+INSERT GRAPHIC HERE ABOUT THE RELATIONSHIPS BETWEEN SYSTEMS/RESOURCES
+PROJECTS/GROUPS PORTFOLIOS PIS USERS
+
+
+Applying for a user account
+===========================
+
+Collaborators involved with an approved and activated NOAA RDHPCS
+project can apply for a user account associated with it. There are
+several steps in receiving a user account, and we're here to help you
+through them.
+
+.. note::
+    Project PIs do not receive a user account with project creation,
+    and must apply for user account to access the NOAA RDHPCS.
+
+
+.. image:: /images/access1.png
+
+Once you have a NOAA.gov email address, you can request an RDHPCS
```

Comment:
Should this be lowercase `noaa.gov` to align to references elsewhere?

Reply:
The original writing had this NOAA.gov usage in an attempt to make it a bit more obvious that all communications happened through their @noaa.gov email account.  I've just stuck with it for the time being.